config/manager/koordlet.yaml

```diff
@@ -23,8 +23,6 @@ spec:
       containers:
         - args:
             - -cgroup-root-dir=/host-cgroup/
-            - -feature-gates=AllAlpha=true
```

Comment:
Why should we remove this? It is to make sure all feature-gates open in e2e or test cluster so that we can test them all.

Reply:
some GA features maybe closed by default, `AllAlpha=true/AllBeta=true` cannot cover this.
how about set the e2e config to open All feature-gates, and use default value here.

Reply:
Here is the YAMLs for test and e2e, so how about set the config to open all features here?

Reply:
sure, -runtime-hooks have been merged into feature-gates